src/main/java/com/wootecobook/turkey/comment/service/dto/CommentResponse.java

```diff
@@ -0,0 +1,46 @@
+package com.wootecobook.turkey.comment.service.dto;
+
+
+import com.wootecobook.turkey.comment.domain.Comment;
+import com.wootecobook.turkey.user.service.dto.UserResponse;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.ToString;
+
+import java.time.LocalDateTime;
+
+@Getter
+@ToString
+@NoArgsConstructor
+public class CommentResponse {
+    private Long id;
+    private Long parentId;
+    private String contents;
+    private UserResponse userResponse;
+    private LocalDateTime createdAt;
+    private LocalDateTime updatedAt;
+
+    @Builder
+    public CommentResponse(final Long id, final Long parentId, final String contents,
+                           final UserResponse userResponse, final LocalDateTime createdAt, final LocalDateTime updatedAt) {
+        this.id = id;
+        this.parentId = parentId;
+        this.contents = contents;
+        this.userResponse = userResponse;
+        this.createdAt = createdAt;
+        this.updatedAt = updatedAt;
+    }
+
+    public static CommentResponse from(Comment comment) {
+        final Long parentId = comment.getParent() == null ? null : comment.getParent().getId();
```

Comment:
Comment 객체에 parent의 id를 가져오는 메서드를 만들어서 활용하는거는 어떤가요??
```suggestion
        final Long parentId = comment.getParentId();
        
        // Comment.java
        class Comment {
            // ...
            public Long getParentId() {
                return comment.getParent() == null ? null : comment.getParent().getId();
            }
        }
```

Reply:
훨씬 좋네요 ! 왜 이생각을 못했지?? 싶네요 감사합니다 ㅎㅎ